packages/components/src/Filter/FilterBar/subcomponents/FilterBarMultiSelect/FilterBarMultiSelect.tsx

```diff
@@ -4,21 +4,19 @@ import {
   FilterMultiSelect,
   FilterMultiSelectProps,
   ItemType,
+  MenuTriggerProviderContextType,
   getSelectedOptionLabels,
 } from "~components/Filter/FilterMultiSelect"
 import { useFilterBarContext } from "../../context/FilterBarContext"
 import { checkArraysMatch } from "../../utils/checkArraysMatch"
 
 export type FilterBarMultiSelectProps = Omit<
   FilterMultiSelectProps,
-  | "isOpen"
-  | "setIsOpen"
-  | "renderTrigger"
-  | "label"
-  | "selectedKeys"
-  | "trigger"
+  "isOpen" | "selectedKeys" | "label" | "trigger"
 > & {
   id?: string
+  label?: string
+  trigger?: (value?: MenuTriggerProviderContextType) => React.ReactNode
```

Comment:
You should be able to do this :)

```suggestion
  label?: FilterMultiSelectProps["label"]
  trigger?: FilterMultiSelectProps["trigger"]
```